x-pack/plugins/observability_solution/observability_ai_assistant_management/public/routes/components/knowledge_base_tab.tsx

```diff
@@ -105,6 +105,15 @@ export function KnowledgeBaseTab() {
         return null;
       },
     },
+    {
+      name: i18n.translate('xpack.observabilityAiAssistantManagement.kbTab.columns.createdBy', {
+        defaultMessage: 'Created by',
+      }),
+      width: '140px',
+      render: (category: KnowledgeBaseEntryCategory) => {
+        return category.entries[0]?.user?.name;
+      },
+    },
```

Comment:
I'm not sure if this will make sense after I merge my PR, since my PR **fixes** the access control checks so that users should only see their own entries in the management UI (so they won't see entries from other users in the same space, their other entries in another space or the built in entries).
What do you think?

Reply:
If a knowledge base entry is global/public everyone who can access the management should be able to see it regardless if they created it or not, no?